### ISSUE TITLE
Call to undefined method Mollie\Api\Resources\Subscription::update()

### ISSUE THREAD

**Issue by hamzarachidi** [mollie/mollie-api-php#533]
Hi, I'm trying to update subscriptions but it seems that there is a problem, is the update function not available anymore?
Thank you

**Comment by MollieRick** [comment]
Hi @hamzarachidi,

Can you elaborate on this? What do you try? What error do you get?

**Comment by hamzarachidi** [comment]
Thank you for your fast respond @MollieRick 
this is the code I'm using 
               ```
                $key = config('env.MOLLIE_KEY');
                $mollie = new MollieApiClient();
                $mollie->setApiKey($key);
                
                $customer = $mollie->customers->get("cst_7k6jRGy6CT");              
                $subscription = $customer->getSubscription("sub_gGrtyhBmg4");
                $subscription->amount = [
                      "currency" => "EUR",
                      "value" => "25.00",
                ];
                $subscription->startDate = "2021-02-22";
                $subscription->interval = "1 month";
                $result = $subscription->update();
```
I'm trying to update an already created subscription like it's described in the code but I'm getting an exception with this message: Call to undefined method Mollie\Api\Resources\Subscription::update()

**Comment by MollieRick** [comment]
@sandervanhooft Can you check this? :)

**Comment by sandervanhooft** [comment]
Hi @hamzarachidi ,

What package version are you using?

**Comment by hamzarachidi** [comment]
Hi @sandervanhooft 
I'm using mollie-api-php version 2.0

**Comment by MollieRick** [comment]
The update-endpoint is added in version 2.0.8. Which makes it that the function does not exist in your project.
However, I recommend you to update to the lastest version available, which is 2.29.0 now. :)

**Comment by hamzarachidi** [comment]
Since I followed the docs I find out myself with version 2.0.0 . I just have to update the package version and it solves the problem. Thank you guys !
